test/pkg/environment/aws/environment.go

```diff
@@ -15,24 +15,25 @@ limitations under the License.
 package aws
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"testing"
 
 	coretest "sigs.k8s.io/karpenter/pkg/test"
 
+	"github.com/aws/aws-sdk-go-v2/service/fis"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	servicesqs "github.com/aws/aws-sdk-go-v2/service/sqs"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
 	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Shouldn't we drop most of these?

Reply:
They are used later in the file to define the new clients